app/views/notes/_notes.html.erb

```diff
@@ -1,3 +1,4 @@
+<%= stylesheet_link_tag "notes", :media => "all" %>
```

Comment:
@Uzay-G Previously I did'nt added it but this was the reply from @jywarren in the previous pr
![Screenshot 2020-01-16 at 9 34 40 PM](https://user-images.githubusercontent.com/57896905/72541249-313d8d00-38a8-11ea-8d22-d55e5798f1d0.png)

Reply:
Hmm. Well, i made the request because it seemed in the screenshots generated here that the styles were not appearing. I don't know that they're automatically included; we do have other styles that are for only specific pages, i believe?

https://stable.publiclab.org/tag/water-quality shows the way that page is supposed to look:

![image](https://user-images.githubusercontent.com/24359/72549544-e4819400-385e-11ea-9140-70755cf85946.png)

And i think the styles weren't getting loaded. Do you have this running locally and can you confirm? If not, we can check the latest auto-generated screenshots for this PR. Thanks for your patience on this!

Reply:
@jywarren  This is the page that I am getting locally without including the stylesheet link
![Screenshot 2020-01-18 at 3 03 55 PM](https://user-images.githubusercontent.com/57896905/72661735-f3ee1200-3a03-11ea-9821-1a71ff25f546.png)

Reply:
And what happens when you include it?

Reply:
@Uzay-G It remains the same
![Screenshot 2020-01-18 at 4 43 15 PM](https://user-images.githubusercontent.com/57896905/72662811-abd5ec00-3a11-11ea-9706-86b4e51e5cbe.png)

Reply:
Yeah alright so we need to remove the stylesheet link tag